web/src/SidebarAccount.tsx

```diff
@@ -252,30 +186,125 @@ function SidebarAccount(props: SidebarAccountProps) {
       </form>
     </SidebarAccountMenuContent>
   )
+}
+
+export { SidebarMenuContent }
+
+/**
+ * Sets up keyboard shortcuts that depend on the sidebar account block.
+ */
+class SidebarAccountShortcuts extends Component<{
+  toggleShortcutsDialog: () => void
+}> {
+  constructor(props: { toggleShortcutsDialog: () => void }) {
+    super(props)
+    this.onKeydown = this.onKeydown.bind(this)
+  }
+
+  componentDidMount() {
+    document.body.addEventListener("keydown", this.onKeydown)
+  }
+
+  componentWillUnmount() {
+    document.body.removeEventListener("keydown", this.onKeydown)
+  }
+
+  onKeydown(e: KeyboardEvent) {
+    if (e.metaKey || e.altKey || e.ctrlKey || e.isComposing) {
+      return
+    }
+    if (e.key === "?") {
+      this.props.toggleShortcutsDialog()
+      e.preventDefault()
+    }
+  }
+
+  render() {
+    return <span></span>
+  }
+}
+
+function SidebarAccount(props: SidebarAccountProps) {
```

Comment:
nbd but might consider renaming "SidebarAccount" as it's no longer just account info. "SidebarInfo"?

Reply:
the diff is misleading - this isn't new code, this was all here before. i'd be open to a PR to rename things a bit, though not sure we should get too hung up on names right now if this is all changing anyway.